stage1/unit4/objects_and_functions.js

```diff
@@ -0,0 +1,59 @@
+//   Создать объект car, добавить к нему свойство color со значением 'черный'
+let car = {
+    color: 'black',
+    carPower: 245,
+}
+
+//Изменить свойство color объекта car на 'зеленый'
+car.color = 'green'
+
+// В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+car.power = function () {
+     return console.log('Лошадиных сил: ' + this.carPower)
+}
+
+car.power()
+
+// На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения количества принятых груш и яблок
+const appleAmount = 2567
+const pearAmount = 1452
+
+function amountOfFruitsReceived(appleAmount, pearAmount) {
```

Comment:
Please rename function. Functions should start with verb. E.g. `getAmountOfFruits`

Reply:
agreed. changed to verb style